spec/cb/clients/cover_letters_spec.rb

```diff
@@ -67,8 +41,12 @@ module Cb
           stub_request(:put, uri).
               with(:body => post_data.to_json, :headers => headers).
               to_return(:status => 500, :body => error_response.to_json)
-          response = Cb::Clients::CoverLetters.create(name: 'name', text: 'text', oauth_token: 'token')
-          expect(response['errors'][0]).to eq(data)
+          begin
+            Cb.cover_letters.create(name: 'name', text: 'text', oauth_token: 'token')
+            expect(false).to eq(true)
```

Comment:
Wow. This needs a message or something. `expect(false).to eq true` is not going to tell me anything at all, until I go to look at the test more closely and see the `rescue` *beneath* it.

What about "expect to raise error" syntax?